packages/styles/scss/components/carousel/_carousel.scss

```diff
@@ -92,36 +93,48 @@
     justify-content: flex-end;
     align-items: center;
     gap: $spacing-05;
-    margin-top: $spacing-05;
+    margin-block-start: $spacing-05;
   }
 
   .#{$prefix}--btn.#{$prefix}--carousel__navigation__btn {
     padding: 0;
-    min-height: $container-03;
-    width: $container-03;
-    height: $container-03;
+    min-block-size: $container-03;
+    block-size: $container-03;
+    inline-size: $container-03;
 
     svg {
       margin: auto;
     }
   }
+
+  .#{$prefix}--carousel__navigation__status {
+    direction: ltr;
```

Comment:
Is there a reason we're forcing left-to-right here?

Reply:
Yes. This selector is for the newly created `<span>` that wraps the carousel status, which shows the ratio of current slide to total slides, eg. "1 / 5". Fractions like this should show the same regardless of language direction. At least, this is based on my light research.

> In-line fractions are written with the numerator and denominator on the left and right of the fraction slash respectively, e.g. ٢/٧ 2/7.

[Source](https://en.wikipedia.org/wiki/Modern_Arabic_mathematical_notation).

Reply:
Oh, smart!